Spec/Auth.swift

```diff
@@ -3619,5 +3619,331 @@ class Auth : QuickSpec {
+                
+                it("the clientId is the same specified in the JWT token request") {
+                    waitUntil(timeout: testTimeout) { done in
+                        client.connection.once(.connected) { _ in
+                            expect(client.auth.clientId).to(equal(clientId))
+                            done()
+                        }
+                        client.connect()
+                    }
+                }
+            }
+            
+            context("when the token request includes subscribe-only capabilities") {
+                let capability = "{\"\(channelName)\":[\"subscribe\"]}"
+                let options = AblyTests.clientOptions()
+                options.tokenDetails = ARTTokenDetails(token: getJWTToken(capability: capability)!)
+                let client = ARTRealtime(options: options)
+                defer { client.dispose(); client.close() }
+
+                it("fails to publish to a channel with subscribe-only capability") {
+                    waitUntil(timeout: testTimeout) { done in
+                        client.channels.get(channelName).publish(messageName, data: nil, callback: { error in
+                            expect(error?.code).to(equal(90001))
+                            expect(error?.message).to(contain("channel operation failed"))
+                            done()
+                        })
+                    }
+                }
+            }
+        }
+        
+        // RSC1 RSC1a RSC1c RSA3d
+        describe("JWT and rest") {
+            let options = AblyTests.clientOptions()
+            
+            context("when the JWT token embeds an Ably token") {
+                options.tokenDetails = ARTTokenDetails(token: getJWTToken(jwtType: "embedded")!)
+                let client = ARTRest(options: options)
+                
+                it ("pulls stats successfully") {
+                    waitUntil(timeout: testTimeout) { done in
+                        client.stats{ stats, error in
+                            expect(error).to(beNil())
+                            done()
+                        }
+                    }
+                }
+            }
+            
+            context("when the JWT token embeds an Ably token and it is requested as encrypted") {
+                options.tokenDetails = ARTTokenDetails(token: getJWTToken(jwtType: "embedded", encrypted: 1)!)
+                let client = ARTRest(options: options)
+                
+                it ("pulls stats successfully") {
+                    waitUntil(timeout: testTimeout) { done in
+                        client.stats{ stats, error in
+                            expect(error).to(beNil())
+                            done()
+                        }
+                    }
+                }
+            }
+            
+            // RSA4f, RSA8c
+            context("when the JWT token is returned with application/jwt content tye") {
```

Comment:
Small typo, `type` not `tye`

Reply:
See https://github.com/ably/ably-ios/pull/714/commits/7351d465fc5690dfad10464ad6f539223e3ecbb7#diff-b5d84fe9bce26902a7481c2ffec84fffR3911